contracts/v2/sovereignChains/AggOracleManager.sol

```diff
@@ -0,0 +1,307 @@
+// SPDX-License-Identifier: AGPL-3.0
+
+pragma solidity 0.8.28;
+
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable5/access/OwnableUpgradeable.sol";
+import {GlobalExitRootManagerL2SovereignChain} from "./GlobalExitRootManagerL2SovereignChain.sol";
```

Comment:
Not an issue at all but is already imported in the Interface.
- Consider adding `insertGlobalExitRoot` in the `IGlobalExitRootManagerL2SovereignChain` and import the interface
- Or, return address in the immutable instead of`GlobalExitRootManagerL2SovereignChain`

Reply:
This way you will not need to import a contract (GlobalExitRootManagerL2SovereignChain) in an Interface (IAggOracleManager).
I mean from:
`GlobalExitRootManagerL2SovereignChain public immutable globalExitRootManagerL2Sovereign;`
to
`address public immutable globalExitRootManagerL2Sovereign;`